pimpmycause/pimpmycause/templates/profiles/profile.html

```diff
@@ -6,39 +6,93 @@
+                                        <label for="surname"> Surname </label>
+                                        <input type="text" name="surname" placeholder="Surname" value='{{ profile_update_form.surname.value|default_if_none:"" }}' required>
                                     </div>
                                 </div>
-                                <div class="row">
-                                    <div class="medium-6 large-4 small-6 columns">
-                                        <input class="button" type="submit" value="Update">
-                                    </div>                                
+                            </div>
+                            <div class="row">
+                                <h2> Social</h2>
+                            </div>
+                            <div class="row">
+                                <div class="large-6 medium-6 small-12 columns">
+                                    {{ profile_update_form.linkedin.errors }}
+                                    <label for="linkedin"> LinkedIn </label>
+                                    <input type="url" name="linkedin" placeholder="LinkedIn" value='{{ profile_update_form.linkedin.value|default_if_none:"" }}' required>
+
+                                    {{ profile_update_form.twitter.errors }}
+                                    <label for="twitter"> Twitter </label>
+                                    <input type="url" name="twitter" placeholder="Twitter" value='{{ profile_update_form.twitter.value|default_if_none:"" }}' required>
+
+                                    {{ profile_update_form.website.errors }}
+                                    <label for="website"> Website </label>
+                                    <input type="url" name="website" placeholder="www.example.com" value='{{profile_update_form.website.value|default_if_none:""}}' required>
+                                </div>
+                                <div class="large-6 medium-6 small-12 columns">
+                                    {{ profile_update_form.phone.errors }}
+                                    <label for="phone"> Phone </label>
+                                    <input type="text" name="phone" placeholder="Phone" value='{{ profile_update_form.phone.value|default_if_none:"" }}' required>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <h2>Experience</h2>
+                            </div>
+                            <div class="row">
+                                <div class="large-12 medium-12 small-12 columns">
+                                        {{ profile_update_form.bio.errors }}
+                                        <label for="bio"> Bio </label>
+                                        <textarea name="bio" rows="20" required placeholder="Write a short bio">
```

Comment:
managed to update my profile without bio details - also couldn't see the placeholder (maybe i got that wrong so let me know please) - also the cursor doesn't start from the top left corner in the text area..
![image](https://cloud.githubusercontent.com/assets/13818956/18889467/014cca1c-84f5-11e6-857b-c3c0df43b91e.png)